fastly/block_fastly_service_v1_acl.go

```diff
@@ -72,6 +82,16 @@ func (h *ACLServiceAttributeHandler) Process(d *schema.ResourceData, latestVersi
 			return err
 		}
 	}
+
+	// UPDATE modified resources (NOT IMPLEMENTED)
+	//
+	// Although the go-fastly API client enables updating of a resource by
+	// its 'name' attribute, this isn't possible within terraform due to
+	// constraints in the data model/schema of the resources not having a uid.
+	//
+	// Because of this we do not implement any logic for updating the dictionary
+	// resource, only CREATE and DELETE functionality.
```

Comment:
As a temporary measure (until we get all `TypeSet` replaced with `TypeList`) I've updated the documentation to reflect that updating the `name` attribute will result in the resource being DELETED and then CREATED (thus losing all data contained within that resource).